File: features/step_definitions/google-search-steps.rb
When /I search Google for "(.*)"/i do |value|
  @browser.get("https://google.com")
  @google_search.search_input.send_keys(value)
  @browser.action.send_keys(:enter).perform
end

Then /I should see "(.*)" in the result/i do |value|
  @browser.find_element(tag_name: 'body').text.should include("#{value}")
end

File: step-definitions/google-search-steps.js
const expect = require('chai').expect;



module.exports = function () {
    this.When(/^I search Google for "([^"]*)"$/, (text) =>{
        return helpers.loadPage('https://google.com')
            .then( () => {
                return page.googleSearch.performSearch(text)
            })
    })
    this.Then(/^I should see "([^"]*)" in the result$/, function (keywords) {
        return driver.wait(until.elementsLocated(by.partialLinkText(keywords)), 10000);
    })
};


